src/main/java/org/jenkinsci/plugins/fodupload/controllers/StaticScanController.java

```diff
@@ -81,9 +82,10 @@ public boolean startStaticScan(final JobModel uploadRequest, final String notes)
             int auditPreferenceId = uploadRequest.isExpressAuditOverride() ? EXPRESS_AUDIT_PREFERENCE_ID : token.getAuditPreferenceId();
 
             Properties props = new Properties();     // TODO Remove test code
-            props.load(this.getClass().getResourceAsStream("/application.properties"));   // TODO Remove test code
+            InputStream inputStream = this.getClass().getResourceAsStream("/application.properties");
+            props.load(inputStream);   // TODO Remove test code
             String projectVersionAttempt4 = props.getProperty("application.version", "NotFound");
-
+            inputStream.close();
```

Comment:
You should use the "try-with-resources" pattern (see https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html) to ensure that the stream is closed even if an error occurs (similar to C# `using` with `IDisposable`). For example, something like this:

```java
try (InputStream inputStream = this.getClass().getResourceAsStream("/application.properties")) {
    props.load(inputStream);
    // use loaded properties
}
```

Reply:
Done. Thanks Kenny. Using/try-resource statements are cleaner than manually disposing of resources.